changelog/7496.improvement.md

```diff
@@ -0,0 +1,35 @@
+Make TED an end-to-end policy. Namely, make it possible to train TED on stories that contain
```

Comment:
Should we maybe add a sentence that TED can still be used in the "old" way and that nothing changed for that?